internal/commands/create.go

```diff
@@ -248,15 +249,39 @@ func getLibraryPath(path string) (string, error) {
 	return libraryPath, nil
 }
 
-func getMatchingLibraries(policy rego.File, libraries []rego.File) []string {
-	var libs []string
-	for _, importPackage := range policy.ImportPackages {
-		for _, library := range libraries {
-			if importPackage == library.PackageName {
-				libs = append(libs, library.Contents)
+func getImportedLibraries(file rego.File, libraries []rego.File) []rego.File {
+	var libs []rego.File
+	for _, i := range file.ImportPackages {
+		for _, l := range libraries {
+			if l.PackageName == i {
```

Comment:
opinion: I tend to reserve single character identifiers for indexing, and use a more descriptive name for the value. I think this comparison takes a little bit of looking up and down to get the full picture of what's happening.